### PR TITLE
squash poll results counter every 10000 instead of 100 to speed up th…

### PR DESCRIPTION
**Pull request by norkans7**
…e sync

### CODE REVIEW

File: ureport/polls/migrations/0037_install_poll_results_count_triggers.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+END;
+$$ LANGUAGE plpgsql;
+-----------------------------------------------------------------------------
+-- Every 10000 inserts or so this will squash the counters by gathering
+-----------------------------------------------------------------------------
+CREATE OR REPLACE FUNCTION
+  ureport_maybe_squash_resultscounters(_org_id INT, _ruleset CHAR(36), _type VARCHAR)
+RETURNS VOID AS $$
+BEGIN
+  IF RANDOM() < .0001 THEN
```
Comment (norkans7) [inline ureport/polls/migrations/0037_install_poll_results_count_triggers.py]: change to squash every 10000 inserts instead of 100

**Comment by rowanseymour** [comment]
Have you thought about putting trigger sql in separate .sql files like rapidpro and casepro? Does make it a little easier to keep track of trigger stuff.

Can't see from diff but is only thing to change here the squash frequency?

**Comment by norkans7** [comment]
Yep the frequency is the only thing changed from https://github.com/rapidpro/ureport/blob/master/ureport/polls/migrations/0032_install_poll_results_count_triggers.py#L27

I can change to use .sql files as well

**Comment by nicpottier** [comment]
This is the only change you've found to speed up the writes? What are the timings on the various pieces of the triggers?

**Comment by norkans7** [comment]
Please review

**Comment by nicpottier** [comment]
Seems good, curious to see what the performance is of squashing 250 runs at a time..

**Comment by nicpottier** [comment]
do it!

**Pull request merged**